src/backend/web/routes/index.js

```diff
@@ -44,4 +44,16 @@ router.use('/stats', stats);
 router.use('/user', user);
 router.use('/query', query);
 
+/**
+ * 404 Handler, Pass to front-end
+ * Leverage .status because adding the `404` status in redirect causes "Not Found. Redirecting to /404?search=" to display.
+ */
+router.use('*', (req, res) => {
```

Comment:
The way this is usually done is as error handling middleware at the end of the routing chain, with an extra error handler at the end:

```js
// Last in the chain
app.use(function (req, res, next) {
  res.status(404).send('...');
});

// Error handling middleware
app.use(function (err, req, res, next) {
  logger.error({ error: err }, 'express web server error');
  res.status(500).send('...')
});
```